sos/report/plugins/ceph_osd.py

```diff
@@ -103,15 +112,21 @@ def setup(self):
                 "/var/snap/microceph/common/logs/*ceph-osd*.log",
```

Comment:
same as above with `direectory` variable replacement?

Reply:
leaving as is for same reasons as above discussions